test/functoria/errors/run.t

```diff
@@ -26,10 +26,10 @@ Clean does not fail
 Help does not fail
   $ ./test.exe help --man-format=plain
   NAME
-         test - The test application builder
+         test-test - The test application builder
```

Comment:
I'm not totally sure why it's `test-test` here.

Reply:
It seems I completely fucked up the evaluation of `Term.ret`'s `` `Help`` case  during the transition. It seem I should be the person test-expecting cmdliner's outputs…